gnomad/utils/constraint.py

```diff
@@ -132,7 +141,7 @@ def collapse_strand(
 def build_models(
```

Comment:
sorry for the late comment on this also, but after re-reading this a few times, I feel like this function belongs more in a pipeline script (something like `gnomad/constraint/pipeline.py` possibly, similar to how we have some more pipeline sample QC functions within https://github.com/broadinstitute/gnomad_methods/blob/main/gnomad/sample_qc/pipeline.py)

Reply:
Maybe we can move the `build_models()` to `gnomad-constraint/gnomad_constraint/utils/constraint.py`? I can definitely create a new file at `gnomad/constraint/pipeline.py`. My concern is that I can't think about other functions to put into this file, and the purpose of this file is similar to the `constraint.py`.

Reply:
open to suggestions here. what do you think @jkgoodrich?

Reply:
Hmm, I see what you mean @ch-kr. I think I'm OK with it being here because to me it makes sense to have it in `gnomad_methods` and I think if there are not other files in `gnomad/constraint/` other than `constraint.py` and `pipeline.py` there isn't really a need for the `pipeline.py` since I think of those as pulling together the more generalized functions in multiple `sample_qc` or `variant_qc` modules. However, I am also fine with whatever the two of you think is best.

Reply:
works for me, let's keep this function in this script then. we can always revisit in the future if needed